### PR TITLE
MP Host Cancel & MP Event Reset & TextField Auto Empty Fixed

### PR DESCRIPTION
**Pull request by Surskit**

### CODE REVIEW

**Comment by Jurgenvs** [comment]
Do note: In order for this to work the Event_Hosts table in the 'remote.sqlite' database has to be deleted.

File: view/src/cg/group4/view/screen/RewardScreen.java [hunk 1]
```diff
@@ -103,6 +103,7 @@ protected void fillDisplay(final Collection collection) {
         DecimalFormat format = new DecimalFormat("0.##");
         java.util.List<Collectible> list = collection.sort(new RarityComparator());
         cContentTable.clear();
+        System.out.println(list);
```
Comment (NickTUD) [inline view/src/cg/group4/view/screen/RewardScreen.java]: Is the Syso here needed? Maybe try to either use log function or just remove it.

File: game_logic/src/cg/group4/util/timer/Timer.java [hunk 2]
```diff
@@ -272,7 +272,7 @@ public final Subject getTickSubject() {
          * Length definition of one stroll.
          * (5 * 60 seconds = 5 minutes)
          */
-        STROLL(5 * 60),
+        STROLL(2 * 60),
```
Comment (NickTUD) [inline game_logic/src/cg/group4/util/timer/Timer.java]: The stroll now lasts 120 seconds, so either comment is outdated or it should be changed back to 300 seconds.

**Comment by NickTUD** [comment]
Looks fine :+1:

File: view/src/cg/group4/view/screen/RewardScreen.java (hunk 1 above)
Comment (Surskit) [inline view/src/cg/group4/view/screen/RewardScreen.java]: Good catch. Was used for debugging. Removed.

File: game_logic/src/cg/group4/util/timer/Timer.java (hunk 2 above)
Comment (Surskit) [inline game_logic/src/cg/group4/util/timer/Timer.java]: Used for testing, I propose to change this right before the end.

Comment (NickTUD) [inline game_logic/src/cg/group4/util/timer/Timer.java]: :+1:

**Pull request merged**